h2o-docs/src/product/flow/packs/examples/XGBoost_Example.flow

```diff
@@ -0,0 +1,93 @@
+{
+  "version": "1.0.0",
+  "cells": [
+    {
+      "type": "md",
+      "input": "# XGBoost Tutorial\n\nThis tutorial walks you through XGBoost usage in H2O Flow.\n\nThose who have never used H2O before should refer to <a href=\"https://github.com/h2oai/h2o-dev/blob/master/h2o-docs/src/product/flow/README.md\" target=\"_blank\">Getting Started</a> for additional instructions on how to run H2O Flow.\n\n## Getting Started\n\nThis tutorial uses a publicly available data set that can be found at:\n\n<a href=\"https://archive.ics.uci.edu/ml/machine-learning-databases/00296/dataset_diabetes.zip\" target=\"_blank\">https://archive.ics.uci.edu/ml/machine-learning-databases/00296/dataset_diabetes.zip</a>.\n\n\nThe original data are the Diabetes data set made available by UCI Machine Learning repository. They are composed of 101 766 observations and 50 attriutes.\n\nIf you don't have any data of your own to work with, you can find some example datasets at <a href=\"http://data.h2o.ai\" target=\"_blank\">http://data.h2o.ai</a>.\n\n### Importing Data\nBefore creating a model, import data into H2O:\n\n0. Click the **Assist Me!** button (the last button in the row of buttons below the menus). ![Assist Me](https://raw.githubusercontent.com/h2oai/h2o-3/master/h2o-docs/src/product/flow/images/Flow_AssistMeButton.png) \n0. Click the **importFiles** link and enter the file path to the dataset in the **Search** entry field. For this example, the file path is http://s3.amazonaws.com/h2o-public-test-data/smalldata/flow_examples/arrhythmia.csv.gz. \n0. Click the **Add all** link to add the file to the import queue, then click the **Import** button."
```

Comment:
Let's remove this "If you don't have any data of your own to work with, you can find some example datasets at <a href=\"http://data.h2o.ai\" target=\"_blank\">http://data.h2o.ai</a>."  Since that resource is very outdated, and replace with the UCI ML main site: https://archive.ics.uci.edu/ml/index.php

Reply:
Is this correct:
"If you don't have any data of your own to work with, you can find some example datasets at <a href=\"https://archive.ics.uci.edu/ml/index.php\" target="_blank">https://archive.ics.uci.edu/ml/index.php"
Or you want it removed altogether?